go.mod

```diff
@@ -1,8 +1,8 @@
 module fyne.io/systray
 
-go 1.19
+go 1.22
```

Comment:
This is a change that really should be done as a separate thing to introducing a new API. However, we have Go 1.19 as a default because it is the same minimum that Fyne uses.

Reply:
Given that Go > 1.21 no longer wants to compile modules with a newer minimum version, this change would make the module unusable for anyone on Go 1.21 not work for Go 1.19 and 1.20 without issues. Given that we don’t use any APIs newer than Go 1.19, bumping to 1.22 in the module makes little to no sense at the moment.

Reply:
I'll remove that commit. We generally keep our modules updated to avoid security issues that get reported by dependabot, but I guess that doesn't apply here.

Reply:
However, I am of the opinion that a PR should do one thing and not multiple things at once. So in that case, updating a dependency that isn’t necessarily needed for this API to work should be in a separate PR.

Reply:
Agreed. I've removed the commit.